mail.md

```diff
@@ -434,7 +436,7 @@ The `withSwiftMessage` method of the `Mailable` base class allows you to registe
     {
         $this->view('emails.orders.shipped');
 
-        $this->withSwiftMessage(function ($message) {
+        $this->withSymfonyMessage(function ($message) {
```

Comment:
It might be prudent to show the type of the message here so people know which object it's referring to.

Reply:
Do you mean something like this?
```suggestion
        $this->withSymfonyMessage(function (Email $message) {
```

Reply:
Yeah and import the namespace through a `use` statement.

Reply:
I checked in Symfony-mailer branch and what is passed to the Callback is `Illuminate\Mail\Message` which wraps around Symfony Email.

Reply:
I know find the name `withSymfonyMessage` a little bit confusing.

Reply:
@jubeki are you sure? I just checked as well and it's the email instance. The whole idea about this method is to directly interact with the email instance.

Reply:
Oh it seems I kind of looked at the wrong callback. https://github.com/laravel/framework/blob/c8d16545df39de602d61998bdeb9326521e0bd9c/src/Illuminate/Mail/Mailer.php#L261-L268